src/components/employeesCard/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../other/icon';
+
+import styles from './EmployeeCard.module.scss';
+
+// TODO: Return a random icon from a few selected for each role
+// ESLinter didn't like break statements in switch since they "can't" be reached
+const iconMatcher = jobType => {
+  switch (jobType) {
+    case 'backEnd':
+      return 'fa-gears';
+
+    case 'frontEnd':
+      return 'fa-flask';
+
+    case 'summerIntern':
+      return 'fa-graduation-cap';
+
+    default:
+      return 'fa-user-secret';
+  }
+};
+
+const EmployeeCard = props => {
+  const { name, jobTitle, description, image, jobType } = props;
+  return (
+    <article className={styles.EmployeeCard}>
+      <img src={image} alt="Stockphoto of a girl smiling" />
```

Comment:
Se om vi kan få inn en bedre `alt` tag her. f.eks "picture of the employee"  eller lignende